### DISCUSSION TITLE
Can I publish the components of dashboard?

### DISCUSSION THREAD

**Discussion by devrsi0n** [plausible/analytics#1504] [Anything Else]
This project is awesome, thank you so much for keeping shaping it and open source!

I have a question, so I have multiply customers using my product, I want to reuse the components of dashboard to build my page, I don't want to copy the code to my repo directly, can I fork this repo and publish a NPM package in `asset` folder? Or could the owner of this repo publish it directly? I think that would be the best solution to benefit the whole community.

**Comment by ukutaht** [comment]
Hi @devrsi0n. Feel free to fork and publish the code as an NPM package.

**Reply by devrsi0n** [reply]
Thank you so much!
